### PR TITLE
Allow inputting performance indicator to students

### LINKED ISSUE #68: Add performance indicator to students
Use performance indicator that is manually entered to track each student

### PR DESCRIPTION
**Pull request by shittake**
Closes #68

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -48,8 +49,10 @@ public AddCommand parse(String args) throws ParseException {
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Photo photo = ParserUtil.parsePhoto(argMultimap.getValue(PREFIX_PHOTO).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
-        Remark remark = new Remark(); // add command does not allow adding remarks straight away
```
Comment (ARPspoofing) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:51]: Same as below, I think Jiatong recently added this so I think we should leave it. Would it be possible to add that back in?

**Comment by ARPspoofing** [comment]
LGTM! I think we should not remove the things Jiatong added for NULL_PERFORMANCE though

File: src/main/java/seedu/address/model/person/Performance.java [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
 
 /**
- * Represents a Person's remark in the address book.
+ * Represents a Person's performance in the address book.
  * Guarantees: immutable; is always valid
  */
 public class Performance {
-    public static final String NULL_PERFORMANCE = "None yet!";
```
[the remark this answers is not in the document]
Comment (shittake) [reply src/main/java/seedu/address/model/person/Performance.java:10]: Got it, think I accidentally deleted that line. Added it back!

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java (hunk 1 above)
Comment (shittake) [reply src/main/java/seedu/address/logic/parser/AddCommandParser.java:51]: Done too

**Pull request merged**